core/io/resource.h

```diff
@@ -54,6 +54,10 @@ class Resource : public RefCounted {
 	GDCLASS(Resource, RefCounted);
 
 public:
+	enum {
+		NOTIFICATION_RESOURCE_DESERIALIZED = 3200,
```

Comment:
Why not start at 3000?

We already have the 1000s range for WM/VP, 2000s for OS and MainLoop, and I don't see any notifications in the 3000s yet.